CRM/Contribute/BAO/Contribution.php

```diff
@@ -512,14 +513,6 @@ public static function create(&$params, $ids = []) {
       throw $e;
     }
 
-    $params['contribution_id'] = $contribution->id;
```

Comment:
It doesn't look to me that $params is picked up from self::add so maybe need this line still?

Reply:
or maybe it is but i'm just confuzzled

Reply:
I put it back - this is the rc & possibly a live patch so conservative is better